packages/contracts/deployments/engine/V3/studio/mainnet/2024-09-05-deployment-config.ts

```diff
@@ -0,0 +1,174 @@
+// This file is used to configure the deployment of Engine and Engine Flex contracts
+// It is intended to be imported by the batch engine factory deployer by running `deploy:v3-engine:mainnet`, `deploy:v3-engine:staging` or `deploy:v3-engine:dev`.
+
+export const deployNetworkConfiguration = {
+  network: "mainnet",
+  // environment is only used for metadata purposes, and is not used in the deployment process
+  // Please set to "dev", "staging", or "mainnet", arbitrum as appropriate
+  environment: "mainnet",
+  useLedgerSigner: true,
+  // ONLY Gnosis Safe is supported for Studio deployments at this time
+  useGnosisSafe: true,
+  safeAddress: "0x52119BB73Ac8bdbE59aF0EEdFd4E4Ee6887Ed2EA",
+  transactionServiceUrl: "https://safe-transaction-mainnet.safe.global",
+  // set the transaction hash after successful execution of the batch creation
+  transactionHash: "",
+};
+
+export const deployConfigDetailsArray = [
+  {
```

Comment:
please add `productClass: "Studio"` to these (see deployment-config.template.ts)

Reply:
🫡